volatility3/framework/constants/__init__.py

```diff
@@ -40,7 +40,7 @@
 # We use the SemVer 2.0.0 versioning scheme
 VERSION_MAJOR = 2  # Number of releases of the library with a breaking change
 VERSION_MINOR = 0  # Number of changes that only add to the interface
-VERSION_PATCH = 2  # Number of changes that do not change the interface
+VERSION_PATCH = 3  # Number of changes that do not change the interface
```

Comment:
There was a small change in the process of merging the plugins (#689), which resulted in version bump. If the PR is merged, we need to update the version one more time. (`2.0.4`)

Reply:
I have updated it to version patch 4.